easybuild/easyconfigs/m/Mesa/Mesa-19.1.7-GCCcore-8.3.0.eb

```diff
@@ -7,8 +7,6 @@
 # it will try to use the llvmpipe by default. It you want swr, do:
 # GALLIUM_DRIVER=swr
 
-easyblock = 'MesonNinja'
```

Comment:
I would prefer having this in a separate PR too

Reply:
See https://github.com/easybuilders/easybuild-easyconfigs/pull/9764